{
  "vulinfoid": "JVNDB-2021-012397",
  "vulinfodata": {
    "title": "PyWPS および OWSLib における XML 外部エンティティの脆弱性",
    "vulinfodescription": {
      "overview": "PyWPS および OWSLib には、XML 外部エンティティの脆弱性が存在します。"
    },
    "affected": {
      "affecteditem": [
        {
          "name": "Debian",
          "productname": "Debian GNU/Linux",
          "cpe": {
            "text": "cpe:/o:debian:debian_linux",
            "version": "2.2"
          }
        },
        {
          "name": "Open Source Geospatial Foundation",
          "productname": "OWSLib",
          "cpe": {
            "text": "cpe:/a:osgeo:owslib",
            "version": "2.2"
          },
          "versionnumber": [
            "0.24.1"
          ]
        },
        {
          "name": "Open Source Geospatial Foundation",
          "productname": "PyWPS",
          "cpe": {
            "text": "cpe:/a:osgeo:pywps",
            "version": "2.2"
          },
          "versionnumber": [
            "4.4.5 未満"
          ]
        }
      ]
    },
    "impact": {
      "cvss": [
        {
          "version": "2.0",
          "severity": {
            "text": "Medium",
            "type": "Base"
          },
          "base": "5",
          "vector": "AV:N/AC:L/Au:N/C:P/I:N/A:N"
        },
        {
          "version": "3.0",
          "severity": {
            "text": "High",
            "type": "Base"
          },
          "base": "7.5",
          "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N"
        }
      ],
      "impactitem": {
        "description": "情報を取得される可能性があります。"
      }
    },
    "solution": {
      "solutionitem": {
        "description": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。"
      }
    },
    "related": {
      "relateditem": [
        {
          "type": "vendor",
          "name": "Debian LTS Advisory",
          "vulinfoid": "[SECURITY] [DLA 2754-1] pywps security update",
          "url": "https://lists.debian.org/debian-lts-announce/2021/09/msg00001.html"
        },
        {
          "type": "vendor",
          "name": "GitHub",
          "vulinfoid": "Propose to replace lxml with defusedxml #790",
          "url": "https://github.com/geopython/OWSLib/issues/790"
        },
        {
          "type": "vendor",
          "name": "GitHub",
          "vulinfoid": "Fix lxml default parser #616",
          "url": "https://github.com/geopython/pywps/pull/616"
        },
        {
          "type": "advisory",
          "name": "Common Vulnerabilities and Exposures (CVE)",
          "vulinfoid": "CVE-2021-39371",
          "url": "https://www.cve.org/CVERecord?id=CVE-2021-39371"
        },
        {
          "type": "advisory",
          "name": "National Vulnerability Database (NVD)",
          "vulinfoid": "CVE-2021-39371",
          "url": "https://nvd.nist.gov/vuln/detail/CVE-2021-39371"
        },
        {
          "type": "cwe",
          "name": "JVNDB",
          "vulinfoid": "CWE-611",
          "url": "https://cwe.mitre.org/data/definitions/611.html",
          "title": "XML 外部エンティティ参照の不適切な制限"
        }
      ]
    },
    "history": {
      "historyitem": [
        {
          "historyno": "1",
          "datetime": "2022-08-30T18:20:29+09:00",
          "description": "[2022年08月30日]\\n  掲載"
        }
      ]
    },
    "datefirstpublished": "2022-08-30T18:20:29+09:00",
    "datelastupdated": "2022-08-30T18:20:29+09:00",
    "datepublic": "2021-07-30T00:00:00+09:00"
  }
}
